core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.xul

```diff
@@ -33,7 +33,8 @@
             <treecol flex="1" label="" primary="true"
                      pen:binding="name" pen:childrenbinding="children"
                      pen:imagebinding="image" editable="false"
-                     pen:classnamebinding="classname" pen:expandedbinding="expanded"/>
+                     pen:classnamebinding="classname" pen:expandedbinding="expanded"
+                     pen:alttextbinding="altText"/>
```

Comment:
this is the glue code that integrates
GwtTreeCole#register
GwtTree#addTreeChild